ga4gh/backend.py

```diff
@@ -740,5 +765,4 @@ def __init__(self, dataDir):
             directory != referencesDirName]
         for datasetDir in datasetDirs:
             dataset = datasets.FileSystemDataset(datasetDir)
-            self._datasetIdMap[dataset.getId()] = dataset
-        self._datasetIds = sorted(self._datasetIdMap.keys())
```

Comment:
I don't think it really matters. There's no requirement in the protocol that these objects be returned in any particular order, so we're not constrained by that. The only issue I can think of potentially causing problems is with paging over multiple processes. If (somehow) two different processes have these lists in different orders, and subsequent page requests get routed to different processes we'll have a bad time.

This seems quite unlikely, but perhaps it's possible. One option is to use the bisect module to insert the ids into the list in sorted order. Another would be to abstract out the file-system walking code and to make sure that files/directories are visited in sorted order. What do you think, is this worth worrying about?